### ISSUE TITLE
startContinuousRecognitionAsync hangs on the first recognized event with the IntentRecognizer

### ISSUE THREAD

**Issue by shinework** [Azure-Samples/cognitive-services-speech-sdk#106]
**Describe the bug**
With the JavaScript SDK, when using the `startContinuousRecognitionAsync` method with a `IntentRecognizer`, after the first  event was recognized, no others events are called. But it works well with the `SpeechRecognizer`.

**To Reproduce**

Speak to the mic: the`recognizer.recognized` listener is called once.

```js
const speechConfig = SpeechSDK.SpeechConfig.fromSubscription(
  LUIS_SUBSCRIPTION_KEY,
  "westeurope"
);

const audioConfig = SpeechSDK.AudioConfig.fromDefaultMicrophoneInput();

let recognizer = new SpeechSDK.IntentRecognizer(speechConfig, audioConfig);
// Note: works well with SpeechSDK.SpeechRecognizer(speechConfig, audioConfig);

recognizer.recognized = (r, event) => {
  console.log(event);
};

recognizer.startContinuousRecognitionAsync();
```

**Expected behavior**
Same behavior as  `SpeechRecognizer`

**Version of the Cognitive Services Speech SDK**
https://github.com/Azure-Samples/cognitive-services-speech-sdk/releases/tag/v1.1.0

**Platform, Operationg System, and Programming Language**
 - OS: MacOS High Sierra 10.13.3 (17D47)
 - x64
 - Version 70.0.3538.102 (Build officiel) (64 bits)

**Comment by zhouwangzw** [comment]
Thank you for reporting the issue.
We are investigating. Sorry for inconvenience.

**Comment by zhouwangzw** [comment]
The issue should be fixed in Release 1.2.0. Please check the latest version.

**Comment by zhouwangzw** [comment]
I am closing the issue. If you still have the issue, please re-open it or open a new one.
